### DISCUSSION TITLE
migrating from an existing aio installation to a (aio) manual-installation

### DISCUSSION THREAD

**Discussion by k4my4b** [nextcloud/all-in-one#2142] [Questions] [answered]
What is the safest way to do so?

**Comment by szaimen** [comment]
Hi, may I ask why you want to do this before I start noting down some steps?

**Reply by k4my4b** [reply]
Hi @szaimen, Primarily for onlyoffice unfortunately collabora/ncoffice is way too lackluster for everyday use I've tried it for at least a couple of months but couldn't cope. Also borg backup is alright but it is way easier for me to backup up/snapshot a couple of data volumes, namely nextcloud_aio_nextcloud_data and nextcloud_aio_database, and it seems more transparent, simpler, fewer moving parts if you know what I mean.

**Comment by szaimen** [comment, marked as answer]
> Primarily for onlyoffice unfortunately collabora/ncoffice is way too lackluster for everyday use I've tried it for at least a couple of months but couldn't cope.

There is still a way to enable OnlyOffice in normal AIO. You may have a look at `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json`.

> backup up/snapshot a couple of data volumes, namely nextcloud_aio_nextcloud_data and nextcloud_aio_database

There are more volumes that should get backed up / snapshotted. You may have a look at https://github.com/nextcloud/all-in-one/tree/main/manual-install#backup-and-restore.

> it seems more transparent, simpler, fewer moving parts if you know what I mean.

I disagree since you are more likely going to make mistakes when doing things incorrectly (like e.g. snapshotting/backing up or updating things incorrectly) but I let you your opinion 😉
